app/views/albums/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h3>Edit Album</h3>
+
+<%= form_for @album, url: {action: "update"}, html: {class: "edit-album"} do |f| %>
```

Comment:
It's been a while so I could be wrong, but I don't think `url: {action: "update"}` is required. Usually the Rails `form_for` can infer whether it's an update or create based on whether `@album` is a new object, or if it's an already-existing object.